### ISSUE TITLE
For multi pose training, the loss does not drop in the early epochs

### ISSUE THREAD

**Issue by liamsun2019** [xingyizhou/CenterNet#898]
As title. Even I apply the pretrained model, the loss does not drop anymore, even rises up, in the very early epochs. How does it happen, any solutions to it?

**Comment by vignesh-debugger** [comment]
Are you training on your custom dataset? or on coco dataset?

**Comment by liamsun2019** [comment]
The original COCO 2017 dataset is applied.  I did another test and used dlav0_34 arch instead. This time the losses seemed to decrease normally although slowly. The training with dla_34 looked that the losses did not decrease anymore in early epoch, i.e, the loss decreased quickly in epoch 0,1, but did not decrease anymore in following epochs, even rised up a little somehow.
